site/zenodo_rdm/moderation/config.py

```diff
@@ -41,21 +41,21 @@
 MODERATION_SPAM_FILE_EXTS = {"jpg", "jpeg", "pdf", "png", "jfif", "docx", "webp"}
 """Frequest spam file extensions."""
 
-MODERATION_RECORD_SCORE_RULES = [
-    verified_user_rule,
-    links_rule,
-    files_rule,
-    text_sanitization_rule,
-    match_query_rule,
-]
+MODERATION_RECORD_SCORE_RULES = {
+    "verified-user-rule": verified_user_rule,
+    "links-rule": links_rule,
+    "files-rule": files_rule,
+    "text-sanitization-rule": text_sanitization_rule,
+    "match-query-rule": match_query_rule,
+}
 """Scoring rules for record moderation."""
 
-MODERATION_COMMUNITY_SCORE_RULES = [
-    links_rule,
-    text_sanitization_rule,
-    verified_user_rule,
-    match_query_rule,
-]
+MODERATION_COMMUNITY_SCORE_RULES = {
+    "links-rule": links_rule,
+    "text-sanitization-rule": text_sanitization_rule,
+    "verified-user-rule": verified_user_rule,
+    "match-query-rule": match_query_rule,
+}
```

Comment:
We can also use class based rules to avoid this and maintaining a sequence in rules.

Reply:
I think it's an easy refactor if we want to achieve that sometime, right now we don't have a sequence based use case.